src/templates/index.hbs

```diff
@@ -1,37 +1,97 @@
-<a href="" onClick="event.preventDefault();bookmarklet.iFrame().init()" class="buttonBookmark">Button Here</a>
+<html>
+<head>
+    <link rel="stylesheet" href="https://cdn.knightlab.com/libs/purpleline/latest/css/purpleline.min.css" type="text/css" />
+    <script type="text/javascript" src="https://cdn.knightlab.com/libs/purpleline/latest/js/purpleline.min.js" ></script>
+       <link rel="icon" type="image/png" href="//cdn.knightlab.com/libs/purpleline/latest/img/favicon.ico">
+</head>
```

Comment:
@shortdiv Yes! I wasn't sure what to do w/default except that it broke the build when I removed it but that totally makes sense--maybe we can add a comment to buildkit or put a note in the readme?